backend/nokia-innovative-project/src/main/java/com/nokia/library/nokiainnovativeproject/services/EmailService.java

```diff
@@ -0,0 +1,51 @@
+package com.nokia.library.nokiainnovativeproject.services;
+
+import com.nokia.library.nokiainnovativeproject.DTOs.EmailDTO;
+import com.nokia.library.nokiainnovativeproject.utils.MessageInfo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.env.Environment;
+import org.springframework.mail.MailException;
+
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.Properties;
+
+@Service
+@RequiredArgsConstructor
+public class EmailService {
+
+    private final Environment environment;
+
+    public MessageInfo sendSimpleMessage(EmailDTO emailDTO) {
+        try {
+            SimpleMailMessage message = new SimpleMailMessage();
+            message.setTo(emailDTO.getRecipients().toArray(new String[emailDTO.getRecipients().size()]));
+            message.setSubject(emailDTO.getSubject());
+            message.setText(emailDTO.getMessageContext());
+
+            javaMailSender().send(message);
+        } catch (MailException exception) {
+            exception.printStackTrace();
+            return new MessageInfo(false, exception.toString(), Arrays.asList("Failed to send messages"));
+        }
+        return new MessageInfo(true, "success", Arrays.asList("Email sent successfully"));
+    }
+
+    private JavaMailSender javaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
+        mailSender.setUsername(environment.getProperty("mail.username"));
+        mailSender.setPassword(environment.getProperty("mail.password"));
+        Properties props = mailSender.getJavaMailProperties();
+        props.put("mail.transport.protocol", "smtp");
```

Comment:
Move properties to 'application.yml' 
Read about @Value annotation. 

Also you use SpringBoot so you have auto configuration for JavaMailSender. 
https://www.baeldung.com/spring-email

You can of course use this configuration :)